### PR TITLE
append 2 blank ballots to test deck

### LINKED ISSUE #1712: Augment VxAdmin test decks with 2 blank HMBPs
Augment VxAdmin test decks with 2 blank HMBPs, one of which will be left blank and one of which will be manually filled out to replace a pre-marked HMBP that will be marked as spoiled

Will likely involve updating [this function](https://github.com/votingworks/vxsuite/blob/1f390f2fa53fc19ba211c06dbe9944a434059eaf/frontends/election-manager/src/utils/election.ts#L177) used [here](https://github.com/votingworks/vxsuite/blob/1f390f2fa53fc19ba211c06dbe9944a434059eaf/frontends/election-manager/src/screens/print_test_deck_screen.tsx#L42)

### PR DESCRIPTION
**Pull request by adghayes**
## Overview
Closes #1712 

Adds 2 blank ballots to the end of each precinct test deck.

## Testing Plan 

Updated test deck snapshot in the L&A test flow 1.

## Checklist
- [ ] I have added [logging](https://github.com/votingworks/vxsuite/tree/main/libs/logging) where appropriate to any new user actions, system updates such as file reads or storage writes, or errors introduced.
- [ ] I have added JSDoc comments to any newly introduced exports

- [ ] I have added a screenshot and/or video to this PR to demo the change
- [x] I have added the "user_facing_change" label to this PR to automate an announcement in #machine-product-updates

### CODE REVIEW

**Comment by adghayes** [comment]
Not sure why a couple tests failed on CircleCI. The frontend test passed locally (and failed here due to a timeout) and the integration test seems unrelated.

**Comment by adghayes** [comment]
@arsalansufi, looking at the rest of the tickets for the L&A roadmap item, it looks like we might want to break out the `generateTestDeckBallots` function into helper functions because it's going to grow substantially. If you think that would be helpful, I'll refactor this change.

**Comment by adghayes** [comment]
I think we need different terminology for what we currently call a test deck (pre-filled ballots that represent every ballot option, either HMPB or machine-marked) and what we are putting together, which is an augmented test deck or full pack of precinct materials.

File: frontends/election-manager/src/utils/election.ts [hunk 1]
```diff
@@ -232,6 +234,19 @@ export function generateTestDeckBallots({
         });
       }
     }
+
+    if (numBlanks && numBlanks > 0) {
+      const blankBallotStyle = precinctBallotStyles[0];
+      if (blankBallotStyle) {
```
[the remark this answers is not in the document]
Comment (adghayes) [reply frontends/election-manager/src/utils/election.ts]: I didn't see a Typescript check (although this is super useful to know for future cases), I just wasn't confident that it's not possible to have a precinct without ballot styles. I can't think of a current counterexample and I haven't tested our current behavior, but jurisdictions do have elections that don't use all their precincts and, in those cases, I wonder if they way they upload the definition will still include the precincts.

I still have the check in the refactored code but can definitely remove!

[the remark this answers is not in the document]
Comment (arsalansufi) [reply frontends/election-manager/src/utils/election.ts]: Given the discussion in Slack (thanks for raising the question there!), I agree that it's the right call to keep the check in!

**Review by arsalansufi — APPROVED** [review]
LGTM!

**Pull request merged**